tests/run-test/bug75042-2.phpt

```diff
@@ -1,15 +1,8 @@
 --TEST--
 phpt EXTENSIONS directive with static module
---SKIPIF--
-<?php
-$php = getenv('TEST_PHP_EXECUTABLE');
-if (false === stripos(`$php -n -m`, 'spl')) {
-	die('skip spl is NOT built static');
-}
---EXTENSIONS--
-SPL
```

Comment:
This is a test of the `--EXTENSIONS--` section, so if it is dropped, this test isn't testing anything anymore. For the purposes of the test it should be fine to drop the skipif, but retain the `--EXTENSIONS--\nSPL` section.

Reply:
Oh, sorry, I drop because I didn't found in [QA](http://qa.php.net/phpt_details.php). Should we update the documentation?